src/constants.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  'PROXY_STATUS_WORKING': 'working',
```

Comment:
Err, maybe not. Symbols allow non-colliding names, but I guess that it's not like the keys here were going to collide anyway. Nevermind haha